### DISCUSSION TITLE
Compiling SCSS into CSS rather than scss.liquid

### DISCUSSION THREAD

**Discussion by jpjmortimer** [hayes0724/shopify-packer#40] [General]
This setup is amazing. Is there a way to get it to compile scss into css rather than getting shopify to compile the scss?

**Comment by hayes0724** [comment]
Several ways, the default config for packer already does this. It creates two output style files:

1. layout.theme.css.liquid (uses theme.scss)
2. layout.theme.styleLiquid.scss.liquid (uses theme.scss.liquid)

The first is compiled css, the second is uncompiled scss for any styles that require liquid.

If you want to make any changes to how packer works, it's easy to extend or override. Use dev.config.js and prod.config.js to change default settings, here is the production config as an example.

[Production Webpack Config](https://github.com/hayes0724/shopify-packer/blob/master/src/webpack/config/prod.config.js)

Packer runs [Webpack Merge](https://www.npmjs.com/package/webpack-merge) to combine configs. You can also modify paths and all other settings inside [packer.config.js](https://github.com/hayes0724/shopify-packer/tree/master/src/config)

**Reply by jpjmortimer** [reply]
Thanks for such a swift reply! I've notice that the layout.theme.css.liquid is created on 'build' but not on 'watch'. I'll try and figure out how to add that to the watch command.

**Reply by hayes0724** [reply]
**Watch** - this uses dev settings + server so the file is stored in memory and should not be visible in dist folder.

Webpack middleware setting: [writeToDisk](https://github.com/webpack/webpack-dev-middleware#writetodisk)

Packer file where this is set: [app.js](https://github.com/hayes0724/shopify-packer/blob/master/src/server/app.js)

**Build** - this uses production config and all files are written to disk (previous files in dist are removed)

**Reply by jpjmortimer** [reply]
OK! Strangely in watch mode the head doesn't pull in layout.theme.css

But after a deploy it does the head will display:
<link type="text/css" href="//cdn.shopify.com/s/files/1/0524/0641/9644/t/2/assets/layout.theme.css?v=6187075895773751665" rel="stylesheet">
and 
<link type="text/css" href="//cdn.shopify.com/s/files/1/0524/0641/9644/t/2/assets/layout.theme.styleLiquid.scss.css?v=5726613116547244338" rel="stylesheet">

So any styles I write to theme.scss doesn't get previewed?
